expand_vocab.ipynb

```diff
@@ -117,10 +117,52 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 5,
```

Comment:
For future PRs, can you clear the output of your notebook?

Reply:
Removed outputs from the notebook.